database/sql/V068__executed_protocol_fees.sql

```diff
@@ -0,0 +1,6 @@
+-- Executed protocol fees are stored in the same ordering as the protocol fees in the fee_policies database table.
+-- protocol_fee_tokens and protocol_fee_amounts are arrays of the same length.
+ALTER TABLE order_execution 
+ADD COLUMN protocol_fee_tokens bytea[] NOT NULL DEFAULT '{}', 
+ADD COLUMN protocol_fee_amounts numeric(78,0)[] NOT NULL DEFAULT '{}';
```

Comment:
Was it considered to use a single JSON column?

Reply:
yes, but i don't see any significant advantage

Reply:
Ability not to think about the ordering of 2 arrays.